app/services/playing-track.js

```diff
@@ -0,0 +1,38 @@
+import Service from '@ember/service';
+import { computed } from '@ember/object';
+
+export default Service.extend({
+  currentlyPlayingTrack: null,
+  isTrackPlaying: false,
+  isTrackPaused: false,
+  // audioSource: `https://p.scdn.co/mp3-preview/697c874cdd9cfd73e1265673f389c86ecec31165?cid=774b29d4f13844c495f206cafdad9c86`,
+  audioSource: computed('currentlyPlayingTrack', function() {
```

Comment:
This would be a great place to use [`alias`](https://emberjs.com/api/ember/3.7/functions/@ember%2Fobject%2Fcomputed/alias)

```suggestion
  audioSource: alias('currentlyPlayingTrack.preview_url'),
```